compiler/one-cmds/one-init

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env bash
+''''export SCRIPT_PATH="$(cd "$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")" && pwd)" # '''
+''''export PY_PATH=${SCRIPT_PATH}/venv/bin/python                                       # '''
+''''test -f ${PY_PATH} && exec ${PY_PATH} "$0" "$@"                                     # '''
+''''echo "Error: Virtual environment not found. Please run 'one-prepare-venv' command." # '''
+''''exit 255                                                                            # '''
+
+# Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import copy
+import os
+import sys
+import shutil
+import utils as _utils
+
+
+# NOTE: Why creates own ConfigParser instead of the existing ConfigParser in python?
+# one-init's role is to guide how to use config parser.
+# The existing ConfigParser in python doesn't keep comments while parsing the config file.
+# To keep comments for guiding how to use config file, let's use own ConfigParser.
+class TemplateConfigParser:
```

Comment:
I make new ConfigParser because the existing ConfigParser in python cannot keep comments of template file.